tests/integration/integration-test.py

```diff
@@ -48,15 +41,23 @@ def test_get_account_org_query(self):
             test_cases=self.test_cases['valid_account_org_query'],
             test_assertion=utils.assertion_tests.actual_equals_test_str)
 
-        utils.test_query_request(
-            self, endpoint='account-indexes',
-            resource='Error',
-            response_code=400,
-            param='accountIndexCode',
-            test_cases=self.test_cases['invalid_account_index_query'])
+    # Test bad request /account-indexes with account index filter
+    # and /account-indexes with organization filter
+    def test_bad_request_get_account_query(self):
```

Comment:
@tsoliangwu0130 recommended more specific function names. For example, this test could be called "test_get_account_query_invalid_parameters", as a bad request could be caused by more than just invalid query parameters.